src/config/reader.rs

```diff
@@ -205,11 +195,10 @@ impl ConfigReader {
             let schema = &file.content;
 
             // Create initial config module
+            let config_path = PathBuf::from(&file.path); // path to config
+            let parent_dir = config_path.parent();
```

Comment:
```suggestion
            let parent_dir = Path::new(&file.path).parent();
```
will it work?

Reply:
`temporary value dropped while borrowed`

Reply:
maybe `Path::new(&file.path).parent()` should work